### PR TITLE
Upgrade code to use it with last version of EasyRdf (namespaced)

### PR DESCRIPTION
**Pull request by fliak**
This is untimely, but it may be helpful for someone, who already have namespaced EasyRdf in project.
Just for share.
@dunglas

### CODE REVIEW

File: src/CardinalitiesExtractor.php [hunk 1]
```diff
@@ -8,6 +8,8 @@
  */
 
 namespace ApiPlatform\SchemaGenerator;
```
Comment (dunglas) [inline src/CardinalitiesExtractor.php]: Missing empty line.

**Comment by dunglas** [comment]
Looks good to me but I'll wait for a new stable release of the EasyRDF library before merging this PR.

File: src/Command/GenerateTypesCommand.php [hunk 2]
```diff
@@ -65,8 +66,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $graphs = [];
         foreach ($processedConfiguration['rdfa'] as $rdfa) {
-            $graph = new \EasyRdf_Graph();
+            $graph = new Graph();
             if ('http://' === substr($rdfa, 0, 7) || 'https://' === substr($rdfa, 0, 8)) {
+//                var_dump($rdfa);
```
Comment (soyuka) [inline src/Command/GenerateTypesCommand.php]: also need to remove

**Comment by k00ni** [comment]
Hey, i know this issue is quite old, but maybe your are still interested in this.

@dunglas wrote:

> Looks good to me but I'll wait for a new stable release of the EasyRDF library before merging this PR.

I started a fork of EasyRdf to allow further maintenance. Further information can be found in https://github.com/njh/easyrdf/issues/320

A first stable release with all changes since EasyRdf 0.9.1 and further fixes and improvements is on the way and available as alpha, if you are interested. Link: https://github.com/sweetyrdf/easyrdf/issues/2

**Comment by dunglas** [comment]
@k00ni for the record I'm 👍 to move to sweetyrdf/easyrdf#2. Don't hesitate to open a PR!

**Comment by k00ni** [comment]
@dunglas that is nice to hear!

As you probably already read, @njh started working on EasyRdf again since 2 weeks and moved the repository to the "easyrdf" organization. He wants to finish the 1.0-release on his own first. Afterwards me and others are welcomed to join as PR approvers.

Therefore i am hesitant. Nevertheless i will keep my fork updated for now and may create a PR here in the future, if EasyRdf got stuck again.

**Comment by dunglas** [comment]
I just finished to read the whole thread, it's nice to see that this library is maintained again! Regarding this lib, we're ok with both solutions as long a we can rely on a maintained and active dependency.

**Comment by dunglas** [comment]
I finally had to upgrade to the upcoming 1.0 (namespaced) version of EasyRDF in #198.

Thanks for your contribution @fliak, it's very appreciated.

**Pull request closed without merging**